.gitignore

```diff
@@ -2,7 +2,7 @@
 # @see https://git.drupalcode.org/project/drupal/-/blob/9.2.x/example.gitignore
 /docroot/core
 /vendor/
-/docroot/sites/*/settings*.php
+/docroot/sites/*/settings.*.php
```

Comment:
@grasmash what's the purpose of this? Note the file comment above, we take this from Drupal core, so if something's wrong it probably needs to get fixed in core: https://git.drupalcode.org/project/drupal/-/blob/9.2.x/example.gitignore

Reply:
With the code as is, settings.php won't be committed. It need to be committed on Acquia hosting.

Reply:
We discussed on a call. It seems like Drupal core intends to ignore settings.php because it might contain db creds. But it _has_ to exist for the site to function, so it needs to be committed (unless there's a way for it to be scaffolded, which I'm not aware of). This seems like a gap in the Drupal and/or Acquia documentation. I opened https://www.drupal.org/project/drupal/issues/3257119

I'm happy to merge this until that issue is addressed.

Reply:
Copying the upstream response: this is only a problem for read-only filesystems, so it's fairly specific to Acquia.